libdrgn/language_c.c

```diff
@@ -1656,9 +1657,15 @@ enum {
 
 #include "c_keywords.inc"
 
+struct drgn_c_family_lexer {
+	struct drgn_lexer lexer;
+	bool cpp;
+};
+
 struct drgn_error *drgn_lexer_c(struct drgn_lexer *lexer,
```

Comment:
There are a couple of other callers of this function that are now accessing invalid memory since they're being given a normal `drgn_lexer`, not the full `drgn_c_family_lexer`. `c_parse_specifier_list` can just hardcode `cpp = false`, and `c_bit_offset` can be given the same treatment as `c_find_type` (add a `lang` parameter and rename it to `c_family_bit_offset`). I'd suggest also renaming this function to `drgn_c_family_lexer_func`. The name is more accurate for its new meaning and renaming it will catch anywhere else we missed.

Reply:
thanks! There is one more place we need to fix in drgn_test_lexer_init. Let me check it